### ISSUE TITLE
Browser time: "Error: PageIndex out of range"

### ISSUE THREAD

**Issue by TheLonelyDev** [sitespeedio/sitespeed.io#2953]
Salutations,

We are currently using SitespeedIO for generating metrics of some sites. When running this tool we have seen an error pop-up on one of the sites we tried this out on. However, we also tested this tool on another ~40 sites and they did not suffer from said error.

I have looked around other bug reports but they do not seem to applicable to this instance since we are not using any custom scripts etc and we are just using Sitespeedio out of the box with Docker.

Thank you for your time in advance, if any information is missing please let me know.

Stacktrace:
```
Could not find the right index 0 for har for url https://lez.dieteren.be
[2020-04-06 14:22:45] ERROR: Error: PageIndex out of range
    at module.exports.pickAPage (/usr/src/app/node_modules/coach-core/lib/har/harCutter.js:18:11)
    at Object.pickAPage (/usr/src/app/node_modules/coach-core/lib/index.js:83:12)
    at Object.processMessage (/usr/src/app/lib/plugins/browsertime/index.js:230:29)
```

Sites with the error: 
- https://lez.dieteren.be
- http://vdfinconditions.be/

- Exact how you run (the exact parameters)
We use SitespeedIO in a bash script with a Docker container, this is how it is being called
`docker run --shm-size=1g --rm -v $(pwd):/sitespeed.io sitespeedio/sitespeed.io:12.4.0-plus1 $url --config config.json`

config.json
```
{
    "browsertime": {
        "iterations": 1,
        "browser": "chrome"
    },
    "graphite": {
        "host": "-redacted-"
    },
    "resultBaseURL": "-redacted-",
    "s3": {
        "key": "-redacted-",
        "secret": "-redacted-",
        "bucketname": "-redacted-",
        "removeLocalResult": true
    },
    "webpagetest": {
        "host": "-redacted-",
        "location": "Win_Client:Chrome",
        "runs": 1,
        "timeout": 6000
    }
}
```

- Which version you are using
Using the current latest version from Docker, 12.4.0-plus1. (This is a headless environment, if this helps out).

- OS (if you use Docker, report the base OS)
```
$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 18.04.4 LTS
Release:        18.04
Codename:       bionic
```

Edit:
- Added another site which is also suffering from the same issue/error

**Comment by soulgalore** [comment]
Hi @TheLonelyDev thanks for the report, at the moment that URL work for me, but I wonder if the have splash screen or some kind of redirect that sometimes happens that we don't handle correctly. Can you see if happens all the time for you or just sometimes?

**Comment by TheLonelyDev** [comment]
I have just ran the test ~5 times again (on the 12.4.0-plus1 & non plus1 image) and another ~5 times on the newly released 12.5.0-plus1 image (also on the non plus1) and I keep getting the same error every time sadly enough.

Some more information regarding to the environment:
`Versions OS: linux 5.0.0-1032-azure nodejs: v12.13.0 sitespeed.io: 12.4.0 browsertime: 8.4.0 coach: 5.0.1` (12.4.0)
&
`Versions OS: linux 5.0.0-1032-azure nodejs: v12.13.0 sitespeed.io: 12.5.0 browsertime: 8.5.0 coach: 5.0.1` (12.5.0)

I have also used debug mode (`-vvv`) and [uploaded the output](https://gist.github.com/TheLonelyDev/a0b1b5d83626454b5380cd29e1e5ff28), hopefully it can provide some closer information.

I have tried the different language pages too (to avoid any redirects) but this does not seem to do anything related to the error that's occurring.

Thank you for your time!
